### DISCUSSION TITLE
Stripe FDW Query Table

### DISCUSSION THREAD

**Discussion by keithkelly31** [supabase/supabase#16474] [Questions]
I created a foreign data wrapper for Stripe through the UI. I created a foreign table to store the subscription data when setting up the wrapper. The table populates properly, but I can't query it.

When I try to run a query, I receive the error "permission denied for schema vault." My vault only contains the Stripe secret. I don't know why my query is attempting to access the vault.

**Comment by MitchellSimY** [comment]
Any update? Having the same issue right now.
